packs/st2cd/actions/git_branch.sh

```diff
@@ -6,14 +6,22 @@ DATE=`date +%s`
 VERSION=$2
 BRANCH=$3
 BASE_BRANCH=$4
-FILE="st2common/st2common/__init__.py"
+ST2COMMON_INIT="st2common/st2common/__init__.py"
+ST2CLIENT_INIT="st2client/st2client/__init__.py"
 OUTPUT=/tmp/git-output-$DATE
 
 cd ${REPO}
 OUT=`$GIT checkout ${BASE_BRANCH} -q > $OUTPUT && $GIT pull origin ${BASE_BRANCH} -q > $OUTPUT && $GIT checkout -b ${BRANCH} -q`
 if [[ $? == 0 ]]
 then
-  VERSIONOUT=`sed -i -e "s/\(__version__ = \).*/\1'${VERSION}'/" ${FILE}`
+  VERSIONOUT=`sed -i -e "s/\(__version__ = \).*/\1'${VERSION}'/" ${ST2COMMON_INIT}`
+  if [[ $? == 0 ]]
+  then
+    VERSIONOUT=`sed -i -e "s/\(__version__ = \).*/\1'${VERSION}'/" ${ST2CLIENT_INIT}`
+  else
+    echo ${OUTPUT}
```

Comment:
Why do we echo the output location here and elsewhere we cat it, then remove it?